index.html

```diff
@@ -301,5 +291,30 @@ <h2 class="section2-h2">skills</h2>
         </li>
       </ul>
     </section>
+
+    <!-- footer -->
+
+    <footer>
+      <h1 class="contact_title">Contact me</h1>
+      <p class="contact_paragraph">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it!
+      </p>
+      <form class="form-section">
+        <label for="name">name</label>
+        <input type="text" id="name" placeholder="enter your name" required />
+
+        <label for="email">email</label>
+        <input
+          type="email"
+          id="email"
+          placeholder="enter your email"
+          required
+        />
+        <textarea id="message">Write your message here.</textarea>
+        <button class="contact_button" type="button">Get in Touch</button>
```

Comment:
Please change your `button` element `type` to `submit` instead of the button. The `<button type="submit">` specifies a submit button. Clicking a submit button sends form data to a form-handler on the server.